cli/lib/cli/args.mjs

```diff
@@ -60,3 +60,7 @@ export async function* getURLs( opt ) {
 		}
 	}
 }
+
+export function logURLProgress( opt ) {
+	return ! opt.url && !! opt.file;
```

Comment:
It could be that there is only a single URL in the file. Would that be a problem?

Reply:
I considered that, but it's not really a big deal. Logging too much isn't an issue, but I think it's reasonable to expect that when a file is provided you want to benchmark multiple URLs. Otherwise you could just provide the URL.

Checking whether there's actually one URL in the file would be overly complicated and involve changing how the file contents are parsed, which right now relies on `yield` for more efficient handling. So I think it's not worth going into that.

Reply:
OK, sounds good. Still, it could be useful even when checking a single `--url`. In that case, what about instead of conditionally logging based on whether `--file` is provided, that there be a new `--verbose` option?

Reply:
I would prefer not to introduce that for now, unless needed. Why would the existing logging be useful for a single URL? I agree logging would be useful if there are multiple runs, and that shouldn't be coupled to how many URLs are being benchmarked. But the information of "Benchmarking {url}" doesn't provide any value IMO if it's only a single URL.

Reply:
Right, this only makes sense when showing progress for multiple runs of a single URL.